buildSrc/build.gradle.kts

```diff
@@ -13,3 +13,10 @@ dependencies {
 	implementation("biz.aQute.bnd:biz.aQute.bnd.gradle:5.0.0")
 	implementation("com.github.jengelman.gradle.plugins:shadow:5.2.0")
 }
+
+allprojects {
+	tasks.withType<AbstractArchiveTask>().configureEach {
+		isPreserveFileTimestamps = false
+		isReproducibleFileOrder = true
+	}
+}
```

Comment:
Do we need this here since we're not publishing any jars produced by `buildSrc`?

Reply:
Good point - I included it for completeness as the output jar was picked up by the testing script, will remove.